### DISCUSSION TITLE
My Fight against Surveillance Capitalism

### DISCUSSION THREAD

**Discussion by cspray** [cspray/blog-comments#8] [General]
Comments for [My Fight against Surveillance Capitalism](https://blog.cspray.io/my-fight-against-surveillance-capitalism)

Everybody is expected to adhere to the [Code of Conduct](https://github.com/cspray/blog-comments/blob/master/CODE_OF_CONDUCT.md)

**Comment by ozankiratli** [comment]
I wanted to thank you for this great post. I've been working on this for almost a year now, and had a decent success. 

Some minor comments on a couple of things that can be done:
1. Major invaders: I would add the ISPs like Comcast to major invaders list, and protect my data against them too. For the project that has PiHole, I can suggest using a VPN service that supports OpenVPN and turning your RPi to a router for your internet (with pihole routing is really easy, even though I still use good old dhcpd), where everything is encrypted, and for DNS, I use Cloudflare and Quad9, maybe using those instead of the ISP provided one will help anonymize your connection details and your location. Just a heads up on this: controlling for DNS and IP leaks might be a little bit painful while using DHCP to get your IP from the ISP but it is possible. 
(I also run a Nextcloud server on another RPi, and would be willing to help you by sharing my experiences and my configuration files)
2. Security: I think security and privacy go hand in hand since if one doesn't secure their online identity today they are more vulnerable against ID theft etc., using password managers (whichever fits) should be encouraged in a day where everyone has tens of online accounts all over the place. 
3. I agree with aligning and paying Tutanota and Purism for their ideals, because I believe in the same things. But there are more options around.
a. email service provider: Protonmail, Hushmail, SCRYPTmail, Mailfence are just a couple to mention, I use Tutanota and Protonmail, for decentralizing my data. I really love Tutanota, and also had no issues with Protonmail.
b. computer choice: There might be people who don't want to sacrifice on their hardware selection and warranty at the same time, and they want to incorporate safe online practices. Buying a Windows installed computer (even though there is a software price) and installing a dual boot Linux on it, and never turning Windows on again, might be a viable option for them.

Finally, I'd be glad if you let me know about your experience with your Purism phone, I'm working on Android to make it safer, but it's been really hard.

**Comment by cspray** [comment]
@ozankiratli Thank you for taking the time to read my post and leave feedback. I also appreciate your offer for assistance. As I get around to these tasks I will be sure to reach out.

1. You're absolutely right about Comcast being a huge violator. I didn't give them the attention they nearly deserve because the idea that security and privacy would be at the forefront of my decision making process is still fairly new. I need more time to research and establish a plan for dealing with my lack of ISP options. Thank you for the information to get started in the right direction.

2. Security _is_ really important and your comment has already got me thinking about a follow-on post talking about my day-to-day security practices (including a password manager and 2FA anywhere I can) and things I want to do to improve it.

3. Thanks for the other options. it is glad to see that there are more options available than those I found. I had checked out Protonmail as well but wound up on Tutanota. I do believe that dual-booting Linux on a Windows OEM will lead to a more secure machine but I wanted to keep the focus of the post on spending revenue at companies you want to support. That being said, I totally understand that not everybody can afford Purism prices and, with how broad and hard the privacy fight is, any measurable progress is important.

I will be sure to review the post and edit the language to make it clear that I have purchased a Librem _laptop_ but I have not decided on the Librem phone yet. After truly evaluating my smartphone usage over the last 5 years I do not have hard requirements for my next phone. With my requirements so low I'm also keeping my eye on https://www.linux.com/blog/2019/2/pine64-launch-open-source-phone-laptop-tablet-and-camera. I hope to be able to afford to put my money behind one of these projects but that isn't in the cards at the moment.

Thanks again for your feedback.

**Comment by macuser666** [comment]
Some additional helpful tools
https://github.com/trailofbits/algo Build your own vpn with AdBlockers and dns over https to Cloudflare 1.1.1.1
Bitwarden crossplatform open source password manager 
https://github.com/bitwarden/mobile
And for Ios and macos users I recommend installing 1blocker X
https://1blocker.com/
Use ublock origin on firefox, privacy badger and https everywhere
